config.php

```diff
@@ -223,8 +223,10 @@
 			?>
 			<script>
                 Shazam.su     = <?php echo SUPER_USER; ?>;
+                Shazam.currentUser = <?php echo json_encode(\ExternalModules\ExternalModules::getUsername()); ?>;
```

Comment:
Also - EMs are not supposed to call \ExternalModules methods directly.  In this case, the current user is USER_ID from the redcap_info() method we ran.  So, you could use that, or if you want more details about the user, you can use REDCap::getUserInfo(USER_ID). from the plugin methods...